src/combine.cpp

```diff
@@ -2334,6 +2334,12 @@ static void fix_casts(chunk_t *start)
    }
    paren_close = pc;
 
+   if (last == NULL)
+   {
+      /* \todo can we be sure that last is never NULL here? */
```

Comment:
When I look through the function `fix_casts` it seems to me that it is well possible that last is NULL. The value of last depends on the chunk list which again depends on the input source file. We can't predict what source file a user presents to uncrustify. And we also have to expect that a source file holds invalid C code. Thus in my opinion the Null check is correct, the assert not.